### PR TITLE
Add sample to import n-grams from Google Books dataset

### PR DESCRIPTION
**Pull request by tswast**
The books dataset isn't quite a CSV or TSV: it's its own special thing. It's close enough that we should be able to adapt it into a more general tool later on.

@lesv PTAL

### CODE REVIEW

File: java/dataflow-load-books/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Building and Running
+
+The examples in this repository can be built and executed from the root directory by running:
+
+Once you have followed the general Cloud Dataflow
+[Getting Started](https://cloud.google.com/dataflow/getting-started) instructions, you can execute
+the same pipeline on fully managed resources in Google Cloud Platform:
+
+Either run the quickstart or get cbt and create the helloworld table and cf1 family.
```
Comment (lesv) [inline java/dataflow-load-books/README.md]: link to quickstart instructions.

File: java/dataflow-load-books/pom.xml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+      <groupId>com.google.truth</groupId>
+      <artifactId>truth</artifactId>
+      <version>0.30</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
```
Comment (lesv) [inline java/dataflow-load-books/pom.xml]: Really don't need this.

File: java/dataflow-load-books/pom.xml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>3.0.2</version>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```
Comment (lesv) [inline java/dataflow-load-books/pom.xml]: defiantly don't need this.

File: java/dataflow-load-books/pom.xml [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.6.0</version>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```
Comment (lesv) [inline java/dataflow-load-books/pom.xml]: I suspect you don't need this.

File: java/dataflow-load-books/pom.xml [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigtable</groupId>
+      <artifactId>bigtable-hbase-dataflow</artifactId>
+      <version>0.9.3</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigtable</groupId>
+      <artifactId>bigtable-hbase-1.1</artifactId>
+      <version>0.9.3</version>
```
Comment (tswast) [inline java/dataflow-load-books/pom.xml]: Done.

File: java/dataflow-load-books/pom.xml (hunk 2 above)
Comment (tswast) [inline java/dataflow-load-books/pom.xml]: Removed.

File: java/dataflow-load-books/pom.xml (hunk 3 above)
Comment (tswast) [inline java/dataflow-load-books/pom.xml]: Removed.

File: java/dataflow-load-books/pom.xml (hunk 4 above)
Comment (tswast) [inline java/dataflow-load-books/pom.xml]: Removed.

I think I needed these until I set the parent, but forgot to remove them after I added it.

**Comment by tswast** [comment]
Waiting to merge. Right now this includes both the Dataflow and Bigtable libraries in the pom.xml so that I can read from Bigtable in the integration tests. These are incompatible.

I've switched the Bigtable package to be a test-scope dependency, but that may not be enough.

Waiting on thoughts from @sduskis on how to proceed.

File: java/dataflow-load-books/pom.xml [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud.dataflow</groupId>
+      <artifactId>google-cloud-dataflow-java-sdk-all</artifactId>
+      <version>1.8.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigtable</groupId>
+      <artifactId>bigtable-hbase-dataflow</artifactId>
+      <version>0.9.4</version>
```
Comment (lesv) [inline java/dataflow-load-books/pom.xml]: 0.9.5 +?

File: java/dataflow-load-books/pom.xml [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
+      <version>1.1.33.Fork23</version>
```
Comment (lesv) [inline java/dataflow-load-books/pom.xml]: Fork26 ??

**Comment by lesv** [comment]
LGTM

**Comment by tswast** [comment]
Thanks. Good catch on the dependency versions. I didn't realize I never checked this in.

**Comment by tswast** [comment]
@dpebot merge when green

**Pull request merged**